apps/browser/qml/pages/components/Overlay.qml

```diff
@@ -541,11 +541,18 @@ Shared.Background {
                     height: searchField.height
                 }
 
-                model: BookmarkModel {
+                BookmarkModel {
                     id: bookmarkModel
                     activeUrl: toolBar.url
                 }
 
+                model: BookmarkFilterModel {
+                    id: bookmarkFilterModel
+                    sourceModel: bookmarkModel
+                    maxDisplayedItems: search ? favoriteGrid.columns : bookmarkModel.count
+                    search: searchField.text !== webView.url ? searchField.text : ""
```

Comment:
Would it make more sense to use `searchField.edited ? searchField.text : ""` here instead?